src/target/riscv/riscv-013.c

```diff
@@ -1974,7 +1974,7 @@ static int examine(struct target *target)
 				info->impebreak);
 	}
 
-	if (info->progbufsize < 4 && riscv_enable_virtual) {
+	if (info->progbufsize < 4 && riscv_target_config_allows_hw_translation(target)) {
 		LOG_TARGET_ERROR(target, "set_enable_virtual is not available on this target. It "
```

Comment:
The old command name (`set_enable_virtual`) in the message should be replaced by the new name.